components/formats-gpl/matlab/bfopen.m

```diff
@@ -46,7 +46,7 @@
 %     bioformats_package.jar.
 %
 % For many examples of how to use the bfopen function, please see:
-%     http://trac.openmicroscopy.org.uk/ome/wiki/BioFormats-Matlab
+%     http://www.openmicroscopy.org/site/support/bio-formats5.1/developers/matlab-dev.html
```

Comment:
Yes, this change mirrors the behavior used for instance in https://github.com/openmicroscopy/bioformats/blob/develop/tools/bf.sh#L72 where the code is bound to a `major.minor` version.